proposals/xxxx-package-manager-branch-support.md

```diff
@@ -0,0 +1,48 @@
+# Package Manager Support for branches in the package manifest
+
+* Proposal: [SE-XXXX](xxxx-package-manager-branch-support.md)
+* Author: [Boris Bügling](https://github.com/neonichu)
+* Review Manager: TBD
+* Status: Discussion
+
+## Introduction
+
+This proposal adds enhancements to the package manifest to support development of packages without strict versioning. It is related to "top of tree" development, but described in a separate proposal, because the implementation is entirely orthogonal to it.
+
+## Motivation
+
+The package manager currently supports packages depedencies which are strictly versioned according to semantic versioning. This works well for users of packages, but we still see some use cases where this hinders common development workflows:
```

Comment:
"Dependencies" is misspelled.
Instead of "This works well for users of packages...", I would say something like "This is how a package's dependencies should be specified when that package is released, but this requirement hinders some development workflows: